org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -36,17 +36,29 @@ class StateReader {
     }
 }
 
+// Helper method to convert Wink's 0.0-1.0 brightness scale to a 0-100 scale
+function scaleBrightnessToDim(brightnessValue) {
+    return Math.floor(brightnessValue * 100);
+}
+
+// Helper method to convert a 0-100 scale to Wink's 0.0-1.0 brightness scale
+function scaleDimToBrightness(dimmingValue) {
```

Comment:
I think the name is fine, a simpler name may be scaleBrightness

---
In reply to: [76908941](https://github.com/openT2T/translators/pull/87#discussion_r76908941) [](ancestors = 76908941)